Philips.CodeAnalysis.Common/Categories.cs

```diff
@@ -4,12 +4,13 @@ namespace Philips.CodeAnalysis.Common
 {
 	public static class Categories
 	{
-		public const string Documentation = @"Documentation";
-		public const string Maintainability = @"Maintainability";
-		public const string Naming = @"Naming";
-		public const string Readability = @"Readability";
-		public const string RuntimeFailure = @"Runtime Failure";
-		public const string Security = @"Security";
-		public const string FunctionalProgramming = @"Functional Programming";
+		public const string Documentation = @"Philips Documentation";
```

Comment:
Another analyzer package I looked at recently used "Company.Category" notation. Idk if that was a one-off, or if this is an established pattern.

Reply:
To me there are two choices with handling the word boundaries:
* Philips Functional Programming
* Philips.Functional.Programming

I proposed the first option, but I'm fine with switching to the second option also.

Reply:
Let's go with the first then.